README.md

```diff
@@ -95,9 +95,8 @@ yarn dev
 yarn test
 ```
 
+# Frames
+
+Frames are expected to follow the Open Frames Standard https://github.com/open-frames/standard
 
-# Key Cases Needed for App Release
```

Comment:
Removed this since it's not entirely accurate right now any ways, I think it's good to be open about release blockers when dealing with open source, but since it's not currently accurate it's not worth having it here right now